bundles/framework/layerlist/view/LayerCollapse/LayerCollapsePanel.jsx

```diff
@@ -66,3 +67,27 @@ LayerCollapsePanel.propTypes = {
     mutator: PropTypes.any.isRequired,
     locale: PropTypes.any.isRequired
 };
+
+const comparisonFn = (prevProps, nextProps) => {
+    // expandIcon is something the parent component adds as a context
+    const ignored = ['expandIcon'];
+    const arrayChildCheck = ['selectedLayerIds'];
```

Comment:
Checking selectedLayerIds with for equality using a shallow comparison would result in memoization never to be used since the array object is freshly created by the parent. So, we are comparing the array contents instead.